### DISCUSSION TITLE
Kiosk Mode Questions

### DISCUSSION THREAD

**Discussion by anthony-breceda-talogy** [SafeExamBrowser/seb-win-refactoring#440] [General]
Hello,

Thank you for the help on issue #428. Changing Kiosk mode seems to be working better for what we are trying to accomplish. We had some additional questions though before we proceed with this.

Is "Disable Explorer Shell" meant to be just as secure from an anti-cheat standpoint? Are there any other loopholes or critical differences we should be aware of by changing kiosk mode? We did notice we needed to create a separate machine-level settings file and have done that. If you have any detailed documentation on this setting we would love to be able to look into it.

Thank You,
Anthony

**Comment by dbuechel** [comment]
We generally recommend to use Create New Desktop, as Disable Explorer Shell means that SEB is running on the default Windows desktop as all other regular applications. But if you carefully test the kiosk mode in your scenario, then it generally should be just as safe to use Disable Explorer Shell, you might just need to add a few more applications to the list of prohibited processes.

Please refer to our official documentation for more information about the two kiosk modes: https://safeexambrowser.org/windows/win_usermanual_en.html#SecurityPane.
